### PR TITLE
Implement 2 cards in EXPERT1 cardset

### PR DESCRIPTION
**Pull request by 95D**
Have a good weekend! 
._.) o

### CODE REVIEW

File: Sources/Rosetta/CardSets/Expert1CardsGen.cpp [hunk 1]
```diff
@@ -201,12 +201,38 @@ void Expert1CardsGen::AddWarlockNonCollect(std::map<std::string, Power>& cards)
...
-    (void)cards;
+    Power power;
+
+    // ----------------------------------------- ENCHANTMENT - WARRIOR
+    // [CS1_104e] Rampage - COST:0
+    // - Faction: Neutral, Set: Expert1, Rarity: Common
+    // --------------------------------------------------------
+    // Text: +3/+3
+    // --------------------------------------------------------
+    // --------------------------------------------------------
```
Comment (utilForever) [inline Sources/Rosetta/CardSets/Expert1CardsGen.cpp]: Please remove overlapped line.

File: Sources/Rosetta/CardSets/Expert1CardsGen.cpp [hunk 2]
```diff
@@ -281,6 +307,17 @@ void Expert1CardsGen::AddNeutral(std::map<std::string, Power>& cards)
     power.AddPowerTask(nullptr);
     cards.emplace("CS2_169", power);
 
+    // --------------------------------------- MINION - NEUTRAL
+    // [CS2_181] Injured Blademaster - COST:3 [ATK:4/HP:7]
+    // - Race: Beast, Faction: Horde, Set: Expert1, Rarity: Rare
+    // --------------------------------------------------------
+    // Text: <b>Battlecry:</b> Deal 4 damage to HIMSELF.
+    // --------------------------------------------------------
+    // --------------------------------------------------------
```
Comment (utilForever) [inline Sources/Rosetta/CardSets/Expert1CardsGen.cpp]: Please remove overlapped line.

File: Tests/UnitTests/CardSets/Expert1CardsGenTests.cpp [hunk 3]
```diff
@@ -95,6 +95,56 @@ TEST(MageExpert1Test, CS2_028_Blizzard)
...
+    curPlayer.SetUsedMana(0);
+    opPlayer.SetTotalMana(10);
+    opPlayer.SetUsedMana(0);
+    
+    auto& curField = curPlayer.GetField();
+    const auto card1 = Generic::DrawCard(
+        curPlayer, Cards::GetInstance().FindCardByName("Boulderfist Ogre"));
+    const auto card2 = Generic::DrawCard(
+        curPlayer, Cards::GetInstance().FindCardByName("Rampage"));
+    const auto card3 = Generic::DrawCard(
```
Comment (utilForever) [inline Tests/UnitTests/CardSets/Expert1CardsGenTests.cpp]: I think `card2` is enough to test the power of card.

File: Tests/UnitTests/CardSets/Expert1CardsGenTests.cpp [hunk 4]
```diff
@@ -95,6 +95,56 @@ TEST(MageExpert1Test, CS2_028_Blizzard)
...
+
+    game.Process(curPlayer, PlayCardTask::Minion(card1));
+    EXPECT_EQ(curField.GetMinion(0)->GetAttack(), 6);
+
+    game.Process(curPlayer, PlayCardTask::SpellTarget(card3, curField.GetMinion(0)));
+    EXPECT_EQ(curField.GetMinion(0)->GetAttack(), 6);
+
+    curField.GetMinion(0)->SetDamage(1);
+    game.Process(curPlayer, PlayCardTask::SpellTarget(card2, curField.GetMinion(0)));
+    EXPECT_EQ(curField.GetMinion(0)->GetAttack(), 9);
```
Comment (utilForever) [inline Tests/UnitTests/CardSets/Expert1CardsGenTests.cpp]: You should add code to check the health of minion to test the power of card.

**Comment by utilForever** [comment]
@95D Please check the review. :smiley:

File: Sources/Rosetta/CardSets/Expert1CardsGen.cpp (hunk 1 above)
Comment (utilForever) [reply Sources/Rosetta/CardSets/Expert1CardsGen.cpp]: Resolved.

File: Tests/UnitTests/CardSets/Expert1CardsGenTests.cpp (hunk 3 above)
Comment (utilForever) [reply Tests/UnitTests/CardSets/Expert1CardsGenTests.cpp]: Resolved.

File: Tests/UnitTests/CardSets/Expert1CardsGenTests.cpp (hunk 4 above)
Comment (utilForever) [reply Tests/UnitTests/CardSets/Expert1CardsGenTests.cpp]: Resolved.

File: Sources/Rosetta/CardSets/Expert1CardsGen.cpp (hunk 2 above)
Comment (utilForever) [reply Sources/Rosetta/CardSets/Expert1CardsGen.cpp]: Resolved.

**Comment by utilForever** [comment]
@95D I applied feedbacks by code review. Thanks. :)

**Comment by 95D** [comment]
I am so sorry that didn't check it. 
I didn't know that you reviewed my code when a day ago. 
I check about your review and fixed code so that will consider it.
._.)a

**Pull request merged**